cmd/cluster-node-tuning-operator/main.go

```diff
@@ -321,6 +334,47 @@ func migratePinnedSingleNodeInfraStatus(cfg *rest.Config, scheme *apiruntime.Sch
 	return nil
 }
 
+func setupFeatureGates(ctx context.Context, config *rest.Config, operatorNamespace string) (featuregates.FeatureGate, error) {
+	// TODO is this version correct?
+	missingVersion := "0.0.1-snapshot"
```

Comment:
Looks fine comparing to https://github.com/openshift/cluster-ingress-operator/blob/master/pkg/operator/operator.go#L100-L131